tests/test_project_management.py

```diff
@@ -0,0 +1,222 @@
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for firebase_admin.project_management."""
+
+import json
+
+import pytest
+
+import firebase_admin
+from firebase_admin import project_management
+from tests import testutils
+
+BASE_URL = 'https://firebase.googleapis.com'
+
+TEST_PROJECT_ID = 'hello-world'
+TEST_ANDROID_APP_ID = '1:12345678:android:deadbeef'
+TEST_ANDROID_APP_NAME = 'projects/{0}/androidApps/{1}'.format(TEST_PROJECT_ID, TEST_ANDROID_APP_ID)
+TEST_ANDROID_APP_DISPLAY_NAME = "My Android App"
+TEST_ANDROID_APP_PACKAGE_NAME = 'com.hello.world.android'
+TEST_ANDROID_APP = {
```

Comment:
Some of these constants are referenced only once. Can we just inline them and remove the constants?

Reply:
I find that when I read a deeply nested test constant, I lose track of the structure of the outermost object. As an example, if we inlined `ANDROID_APP_NO_DISPLAY_NAME_OPERATION_SUCCESSFUL_RESPONSE`, in which we inline `TEST_ANDROID_APP_NO_DISPLAY_NAME`, then it's going to make the test `test_create_android_app_without_display_name` responses list (lines 125-127) much harder to digest. Currently, I believe that list is really easy to digest, with each response named and commented and still fitting in one line each.

Reply:
I still think that we are overdoing it with constants. I often read code on vim and emacs with only basic text editing support, and I find myself jumping back and forth a lot trying to determine the full value of some constants. For instance in the above example, can we get rid of the `TEST_ANDROID_APP_NO_DISPAY_NAME` constant by inlining?

Having said that, I don't mind leaving things as is for now, and revising this later.

Reply:
I agree that inlining the simple, unnested constants is probably not going to hurt readability. This will tone down the level we are naming constants, while still making the test shorter and easier to understand for nested constants. I will, however, do most of this work in the next PR, so that my merges are easier.